README.md

```diff
@@ -433,6 +433,60 @@ $contents = file_get_contents('gs://my_bucket/file_backup.txt');
 $ composer require google/cloud-storage
 ```
 
+## Google Cloud Tasks (GA)
+- [API Documentation](http://googleapis.github.io/google-cloud-php/#/docs/latest/tasks/readme)
+
+#### Preview
+
+```php
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Google\Cloud\Tasks\V2\CloudTasksClient;
+use Google\Cloud\Tasks\V2\Queue;
+
+$client = new CloudTasksClient();
+
+$project = 'example-project';
+$location = 'us-central1';
+$queue = uniqid('example-queue-');
+$queueName = $client::queueName($project, $location, $queue);
+
+// Create a queue
+$locationName = $client::locationName($project, $location);
+$queue = new Queue([
+    'name' => $queueName
+]);
+$queue->setName($queueName);
+$client->createQueue($locationName, $queue);
+
+echo "$queueName created." . PHP_EOL;
+
+// List queues
+echo 'Listing the queues' . PHP_EOL;
+$resp = $client->listQueues($locationName);
+foreach ($resp->iterateAllElements() as $q) {
+    echo $q->getName() . PHP_EOL;
+}
+
+// Delete the queue
+$client->deleteQueue($queueName);
+```
+
+#### Removal of pull queue
```

Comment:
Do we need to maintain this notice indefinitely?

Reply:
No, I don't think so. We could probably give it another month or so and then remove it. WDYT?

Reply:
sounds good to me.